[TR] Chat Console (#3550)
ppy/osu-wiki: wiki/Chat_Console/tr.md

<!-- TODO: dated images and links, formatting problems, writing could be done better instead of all these lists. I removed the `needs_cleanup` tag because this still details the Chat Console pretty well. -->

> # Sohbet Konsolu

Reviewer: ```suggestion
# Sohbet konsolu
```

- Lock Panels panellerin etrafta sekip durmasını ve yeni kullanıcıların ekrana düşmesini engeller.
- Kullanıcı panellerini bir özniteliğe göre sıralamak için o sekmeye tıklayın.
- Herkesin nerede olduğunu gösteren bir dünya haritasını görüntülemek için dünya haritasına tıklayın.
- Kaydırmak için beyaz kutuyu tıklatıp sürükleyebilirsiniz, fare tekerleğini de kullanabilirsiniz.
- Panellerinde istatistik olmayan kullanıcılar IRC istemcilerini kullanarak sohbete bağlanmıştır.

| Panel rengi | Açıklama |
| :-- | :-- |
| ![Player is idle](img/Chat_Console-Idle.png "Oyuncu boşta") | Lacivert - Oyuncu şu anda boşta, bir şey yapmıyor ya da sadece sohbet ediyor. |
| ![Player is playing](img/Chat_Console-Playing.png "Oyuncu oyunda") | Gri - Solo'da bir beatmap oynuyor. |
| ![Player is watching](img/Chat_Console-Watching.png "Oyuncu izliyor") | Açık Mavi - Bir tekrarı veya birini izliyor. |
| ![Player is editing a beatmap](img/Chat_Console-Editing.png "Oyuncu bir beatmap düzenliyor") | Kırmızı - Kendine ait bir beatmapi düzenliyor. |
| ![Player is testing a beatmap](img/Chat_Console-Testing.png "Oyuncu bir beatmap test ediyor") | Mor - Kendisinin ya da bir başkasının beatmapini test ediyor. |

> | ![Player is submitting a beatmap](img/Chat_Console-Submitting.png "Oyuncu bir beatmap gönderiyor") | Turkuaz - Yaptıkları bir beatmapi gönderiyor (yükleme veya güncelleme). |

Reviewer: ```suggestion
| ![Player is submitting a beatmap](img/Chat_Console-Submitting.png "Oyuncu bir beatmap gönderiyor") | Turkuaz - Yaptığı bir beatmapi gönderiyor (karşıya yükleme veya güncelleme). |
```
thought this would look better

| ![Player is modding a beatmap](img/Chat_Console-Modding.png "Oyuncu bir beatmap modluyor") | Yeşil - Bir başkasının beatmapini modluyor ya da düzenliyor. |

> | ![Player in Multiplayer](img/Chat_Console-Multiplayer.png "Oyuncu Çoklu'da") | Kahverengi - Kullanıcı Çoklu modda, ama oynamıyor. |

Reviewer: ```suggestion
| ![Player in Multiplayer](img/Chat_Console-Multiplayer.png "Oyuncu Çok oyunculuda") | Kahverengi - Kullanıcı çok oyunculu modda, ama oynamıyor. |
```

> | ![Oyuncu Multiplayer oynuyor](img/Chat_Console-Multiplaying.png "Oyuncu Çoklu'da oynuyor") | Sarı - Şu anda Çoklu'da oynuyor. |

Reviewer: ```suggestion
| ![Oyuncu Multiplayer oynuyor](img/Chat_Console-Multiplaying.png "Oyuncu Çok oyunculuda oynuyor") | Sarı - Şu anda çok oyunculu modda oynuyor. |
```

| ! [Player is AFK](img/Chat_Console-Afk.png "Oyuncu AFK") | Siyah - Aktif değil ya da klavyeden uzakta (AFK). |
| ![Player is connected via IRC](img/Chat_Console-IRC.png "Oyuncu IRC üzerinden bağlıdır") | İçeriksiz Lacivert - Oyuncu oyun içinde değildir, ancak bir IRC istemcisine bağlıdır ya da istatistikler mevcut değildir. |

Herhangi bir kullanıcı paneline tıkladığınızda bir seçenek ekranı açılır.

![User panel options](img/Chat_Console-Userpaneloptions.jpg "Kullanıcı paneli seçenekleri")

Etkinleştirmek için numarayı tıklatın veya çubuğu tıklatın:

1. `İzlemeye Başla`: Eğer kullanıcı bir beatmap oynuyorsa ve siz beatmape sahipseniz, onları oynarken izleyebilirsiniz. Adınız Seyirciler listesinde yer alacak.

> 2. `Profili Göster`: Oyuncunun profilinin Web sayfası tarayıcınızda açılır.

Reviewer: ```suggestion
2. `Profili Göster`: Oyuncunun profil web sayfası tarayıcınızda açılır.
```
thought this would look better

3. `Sohbeti Başlat`: Kullanıcıyla özel sohbet sekmesini açar.
4. `Oyuna davet et`: (Çoklu odadaysanız) Kullanıcıdan odanıza gelmesini isteyin.

> 5. `Arkadaş Olarak Ekle (Arkadaşlıktan Kaldır)`: Kullanıcıyı arkadaş listenize ekler (veya listenizden kaldırır)

Reviewer: ```suggestion
5. `Arkadaş Olarak Ekle (Arkadaşlıktan Çıkar)`: Kullanıcıyı arkadaş listenize ekler (veya listenizden çıkarır)
```
translated this way in-game

6. `Kullanıcıyı Bildir...`: Kullanıcıyı kötü davranış için bildirin. Aksi belirtilmedikçe asla kullanılmamalıdır. Bir kullanıcıyı oyun içinde, ya da forumlarda olduğu gibi web'de, belli birkaç şey için raporlayabilirsiniz,
7. `Kullanıcıyı Sustur`: Kullanıcı tarafından yapılan herhangi bir sohbet gönderisi sizin sohbet konsolunuzda gözükmeyecektir.
8. `Kapat`: Paneli kapatır.

## Komut listesi

### /help

| Komut | Etki | Örnek | BanchoBot yanıtı |
| :-- | :-- | :-- | :-- |
| `/addfriend [kullanıcı]` | Arkadaş listenize `[kullanıcı]` ekleyin. | `/addfriend Amigo` | Artık Amigo ile arkadaşsınız. |
| `/delfriend [kullanıcı]` | Arkadaş listenizden `[kullanıcı]` kaldırın. | `/delfriend Amigo` | Artık Amigo ile arkadaş değilsiniz. |
| `/away [mesaj]` | Bir dışarıda iletisi ayarlar (Size PM gönderen kullanıcılara gönderilir). İptal etmek için iletiyi boş bırakın. | `/away Ben John Smith.` | You have been marked as being away: Ben John Smith. Amigo size /msg John Neredesin~? dediğinde BanchoBot: Ben John Smith. |

> | `/bb` | BanchoBot'a bir komutu doğrudan göndermenizi sağlar. | `/bb !faq tr:peppy` | peppy, osu!'nun kurucusu ve baş geliştiricisidir. Projelerin çoğunu kendisi yönetiyor. |

Reviewer: the English document was written differently here. perhaps we should change this, although `rank #` command doesn't seem to work, but `!stats [user]` do

| `/chat [kullanıcı]` | Belirtilen kullanıcıyla yeni bir sohbet sekmesi açın. | `/chat Amigo` | (Amigo sekmesi açıldı) |

> | `/clear` | Geçerli sohbet arabelleğini temizler. | '/clear' | (Geçerli sekmesinde aşağı yukarı her şeyi temizler) |

Reviewer: ```suggestion
| `/clear` | Geçerli sohbet penceresini temizler. | '/clear' | (Geçerli sekmede aşağı yukarı her şeyi temizler) |
```
let's avoid using complex words
